src/Tests/engine/EntityCheck.cpp

```diff
@@ -1,7 +1,7 @@
 #include <catch.hpp>
 #include <Engine/Entity/Entity.hpp>
 
-TEST_CASE("", "[entity component]")
+TEST_CASE("Entity check", "[entity component]")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'TEST_CASE' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)

Reply:
You are wrong, plz..